OpenSim/Simulation/Model/ContactSphere.h

```diff
@@ -37,11 +36,9 @@ class OSIMSIMULATION_API ContactSphere : public ContactGeometry {
 OpenSim_DECLARE_CONCRETE_OBJECT(ContactSphere, ContactGeometry);
 
 //=============================================================================
-// DATA
+// PROPERTIES
 //=============================================================================
-protected:
-    PropertyDbl _radiusProp;
-    double& _radius;
+    OpenSim_DECLARE_PROPERTY(radius, double, "Radius of the sphere.");
```

Comment:
Not critical, but some property comments mention the default value. Might be particularly helpful in this case because the default value is zero (so the user won't see anything unless the radius has been set).